server/pulp/server/webservices/views/content.py

```diff
@@ -167,7 +170,7 @@ def post(self, request):
 
         :raises: OperationPostponed when an async operation is performed
         """
-        all_orphans = request.body_as_json
+        all_orphans = request.body_as_json or {}
```

Comment:
Do you want this to be treated as an expression? Maybe you want this to act like a ternary operator, but this is going to return True/False based on the or statement. Is that what you want?